### PR TITLE
test(cli): use a more strict check license text

### PR DESCRIPTION
**Pull request by bajtos**
Rework the assertions in `lb4 copyright` tests to verify the entire content of the `LICENSE` file generated, not just isolated substrings.

There is no change in CLI functionality, I am only updating the test to better describe the current behavior.

See also https://github.com/strongloop/loopback-next/pull/7107

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- API Documentation in code was updated
- Documentation in [/docs/site](../tree/master/docs/site) was updated
- Affected artifact templates in `packages/cli` were updated
- Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/cli/test/integration/generators/copyright.integration.js [hunk 1]
```diff
@@ -202,28 +202,36 @@ restricted by GSA ADP Schedule Contract with <%= owner %>.
...
+Node module: myapp
+This project is licensed under the ISC License, full text below.
+
+--------
+
+ISC License
+
+ISC License Copyright (c) 2004-2010 by Internet Systems Consortium, Inc. ("ISC")
+
+Copyright (c) 1995-2003 by Internet Software Consortium
```
Comment (bajtos) [inline packages/cli/test/integration/generators/copyright.integration.js:221]: On this line, I think the copyright holder should be ACME Inc., not Internet Software Consortium. Fixing that is out of scope of this pull request though.

Comment (bajtos) [reply packages/cli/test/integration/generators/copyright.integration.js:221]: See https://opensource.org/licenses/ISC

**Review by raymondfeng — APPROVED** [review]

**Review by dhmlau — APPROVED** [review]

**Review by frbuceta — APPROVED** [review]
